app/src/main/java/org/eyeseetea/malariacare/domain/exception/NetworkException.java

```diff
@@ -0,0 +1,8 @@
+package org.eyeseetea.malariacare.domain.exception;
+
+
+public class NetworkException extends Exception {
+    public NetworkException() {
+        super("Credentials not valid");
```

Comment:
@xurxodev  can you change "Credentials not valid" by "Network error" or similar? thanks